platform/platform_linux.cpp

```diff
@@ -268,8 +268,15 @@ void Platform::GetSystemFontNames(FilesList & res) const
 // static
 time_t Platform::GetFileCreationTime(std::string const & path)
 {
+// In older Linux versions there is no reliable way to get file creation time.
+#if __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 28  
```

Comment:
I wonder if it'll be better to check directly if `statx()` is defined instead of relying on glibc version? Is it possible some glibc replacement like musl, etc. is used?

Reply:
syscall can be used, but that's way too complex. That function is not critical for Linux, and the result of calling it on older Linuxes/file systems without birth time is not reliable anyway. let's compile it now and move on.